docs/source/using_doctr/using_datasets.rst

```diff
@@ -84,6 +86,8 @@ This datasets contains the information to train or validate a text recognition m
 +-----------------------------+---------------------------------+---------------------------------+---------------------------------------------+
 | IIITHWS                     | 7141797                         | 793533                          | english / handwritten / external resources  |
 +-----------------------------+---------------------------------+---------------------------------+---------------------------------------------+
+| WILDRECEIPT                 | 1268                            | 472                             | english / external resources                |
```

Comment:
This looks not correct here we should add the number of samples we get if we use the dataset for recognition :)
So this should be much more